apps/interactor/interactor/commander/commands/clean.py

```diff
@@ -0,0 +1,35 @@
+from interactor.commander.command import DeviceChangeMixin
+from interactor.commander import helpers as ihp
+from interactor.commander.store import store
+
+from photons_control.planner import Skip
+
+from delfick_project.norms import dictobj, sb
+from photons_control.script import FromGeneratorPerSerial
+from photons_messages.messages import LightMessages
+
+
+@store.command(name="clean/status")
+class StatusCleanCommand(store.Command, DeviceChangeMixin):
+    """
+    Returns the current state of an HEV cleaning cycle on supported devices
+    """
+
+    async def execute(self):
+        plans = self.sender.make_plans("capability", "hev_cycle")
+        self.sender.gatherer.clear_cache()
```

Comment:
```suggestion
```
Note necessary to clear the cache. I tend to only use that if I know that I just used it and really need it to refresh

Reply:
I'd put refresh = True on the hev_cycle plan itself in this case